README.md

```diff
@@ -1,39 +1,62 @@
-
 This plugin allows you to launch Firefox profiles from Krunner and your normal launcher.
+It saves the Desktop Actions for the profiles in a local copy of the firefox.desktop file.
+This mean you can search for the profiles in your normal launcher.
+
+- Saves the Desktop Actions for the profiles in a local copy of the `firefox.desktop` file. This mean you can search for the profiles in your normal launcher.
+- Compatible with Firefox-ESR.
+- Has proxychains4 integration. This allows you to launch specific profiles over a proxy. This integration is optional.
 
-The plugin uses a local copy of the firefox.desktop file. In this file it adds Desktop Action entries for
-the profiles. Because of this you can search for the profiles in your normal launcher.  
-This plugin is also compatible with firefox-esr.
+<details>
+<summary><b><b>Required Dependencies</b></b></summary>
+
+<b>Debian/Ubuntu:</b>
+```shell
+sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libkf5kcmutils-dev
+```
 
-Furthermore it provides proxychains4 integration. This allows you to launch specific profiles over a proxy.
-This integration is optional.
+<b>OpenSUSE:</b>
+```shell
+sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel kservice-devel krunner-devel gettext-tools kconfigwidgets-devel kcmutils-devel
+```
 
-You can find deb/rpm packages on the release page.
+<b>Fedora:</b>
+```shell
+sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext kf5-kcmutils-devel
+```
 
-### Required Dependencies
+<b>Arch (Manjaro):</b>
+```shell
+sudo dnf install cmake extra-cmake-modules kcmutils
+```
 
-Debian/Ubuntu  
-`sudo apt install cmake extra-cmake-modules build-essential libkf5runner-dev libkf5textwidgets-dev qtdeclarative5-dev gettext libkf5kcmutils-dev`
+</details>
 
-openSUSE  
-`sudo zypper install cmake extra-cmake-modules libQt5Widgets5 libQt5Core5 libqt5-qtlocation-devel ki18n-devel ktextwidgets-devel kservice-devel krunner-devel gettext-tools kconfigwidgets-devel kcmutils-devel`  
+<details>
+<summary><b>Build Instructions</b></summary>
 
-Fedora  
-`sudo dnf install cmake extra-cmake-modules kf5-ki18n-devel kf5-kservice-devel kf5-krunner-devel kf5-ktextwidgets-devel gettext kf5-kcmutils-devel`  
+<details>
+<summary><b>A: Oneliner install</b></summary>
 
-Arch(Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules kcmutils`  
+```shell
+curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install.sh | bash
+```
 
-### Build instructions
+</details>
 
-The easiest way to install is:  
-`curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install.sh | bash`  
-Or you can install it without admin privileges:  
-`curl https://raw.githubusercontent.com/alex1701c/krunner-firefox/master/install-user.sh | bash`  
+<details>
```

Comment:
(as in the other PR), please do not use the details for these simple install commands

Reply:
This thread needs to be addressed, then it is ready to be merged